packages/ecmascript/src/Set.ts

```diff
@@ -49,17 +49,17 @@ export default class Set<T extends string | number> {
 		return values;
 	}
 
-	has(value): boolean {
+	public has(value: any): boolean {
 		return this.items.hasOwnProperty(value);
 	}
 
-	add(value): any {
+	public add(value: any): any {
 		this.items[value] = true;
 		return this;
 	}
```

Comment:
Don't we need to increment the size property here?

Reply:
Code updated.